### PR TITLE
[CM-1346] - Add support for correct attachment name and icon

### PR DESCRIPTION
**Pull request by amntoppoKM**
## Issue:
1. Attachment name is getting saved with concatenated message timestamp. This is displayed to end-user. If the attachment name is "aman.pdf". It will show to user as "aman1121413213.pdf"
2. Attachment name was having "AWS-ENCRYPTED-" string
3. For attachments, correct application was not showing and it was throwing an error

## Fix:
- Files are saved with message timestamp, to prevent same name files from getting replaced.
- We are removing "AWS-ENCRYPTED" and message timestamp from UI, but not from real filename.
- We have added PDF and Word icons.
- Fixed Intents to open correct type of application depending on mimeType.

image

### CODE REVIEW

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/attachmentview/KmDocumentView.java [hunk 1]
```diff
@@ -187,11 +189,12 @@ public void inflateViewWithMessage(View rootview, Message message) {
                 mimeType = FileUtils.getMimeType(filePath);
                 if (mimeType != null && !(mimeType.contains("audio"))) {
                     String fileName = new File(filePath).getName();
```
Comment (Sathyan-KM) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/attachmentview/KmDocumentView.java:191]: Are we using fileName anywhere ? if no remove it

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/attachmentview/KmDocumentView.java [hunk 2]
```diff
@@ -187,11 +189,12 @@ public void inflateViewWithMessage(View rootview, Message message) {
...
-                    fileText.setText(fileName);
+                    fileText.setText(KmUtils.getAttachmentName(message));
                     audioseekbar.setVisibility(GONE);
                     audio_duration_textView.setVisibility(GONE);
                     docIcon.setVisibility(View.VISIBLE);
-                    docIcon.setImageResource(R.drawable.ic_documentreceive);
+                    setDocumentIcon(mimeType, docIcon);
+//                    docIcon.setImageResource(R.drawable.ic_documentreceive);
```
Comment (Sathyan-KM) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/attachmentview/KmDocumentView.java:197]: remove commented lines

File: kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java [hunk 3]
```diff
@@ -2790,7 +2791,9 @@ public void run() {
                                 if (message.getFileMetas().getContentType().contains("audio")) {
                                     imageViewDoc.setImageResource(R.drawable.ic_play_circle_outline);
                                 } else {
-                                    imageViewDoc.setImageResource(R.drawable.ic_documentreceive);
+                                    setDocumentIcon(null, imageViewDoc);
+                                    setDocumentIcon(message.getFileMetas().getContentType(), imageViewDoc);
+                                   // imageViewDoc.setImageResource(R.drawable.ic_documentreceive);
```
Comment (Sathyan-KM) [inline kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java:2796]: remove commented lines throughout the PR

**Comment by amntoppoKM** [comment]
@Sathyan-KM made the changes

**Review by Sathyan-KM — APPROVED** [review]

**Pull request merged**